actions/admin/paymail_addresses_old.go

```diff
@@ -163,6 +164,15 @@ func paymailCreateAddressOld(c *gin.Context, _ *reqctx.AdminContext) {
 		opts = append(opts, engine.WithMetadatas(requestBody.Metadata))
 	}
 
+	config := reqctx.AppConfig(c)
+	if config.Paymail.DomainValidationEnabled {
+		_, actualDomain, _ := paymail.SanitizePaymail(requestBody.Address)
+		if !checkPaymailDomain(actualDomain, config.Paymail.Domains) {
+			spverrors.ErrorResponse(c, spverrors.ErrInvalidDomain, logger)
+			return
+		}
+	}
```

Comment:
This is duplicated code.
But since we plan to remove the "old" handlers soon (I believe 🙈), it is totally fine to me 👍